integration/singlearch/patch_test.go

```diff
@@ -176,6 +184,11 @@ func patch(t *testing.T, ref, patchedTag, path string, ignoreErrors bool, report
 		reportPath = "-r=" + path + "/scan.json"
 	}
 
+	var platformsFlag string
+	if !reportFile {
+		platformsFlag = "--platforms=linux/amd64"
```

Comment:
are we doing this because tests take too long?

Reply:
I added this because this workflow is a single architecture test so I was thinking we can patch/scan for the workflow platform? it would also increase test time if we wanted to patch all platforms